zipa/magic.py

```diff
@@ -28,6 +28,7 @@ def __getattr__(self, name):
         return self.env[name]
 
     def _parse_name(self, name):
+        name = name.replace('___', '-')
```

Comment:
There are three underscores there, @krodyrobi . Because two underscores are already used for slash representation and one underscore for dot representation, the only available option I saw for representing dashes without getting a parsing error from the interpreter was three underscores.